multiqc/search_patterns.yaml

```diff
@@ -210,17 +210,17 @@ ccs/v5:
 cellranger/count_html:
   - fn: "*.html"
     contents: '"command":"Cell Ranger","subcommand":"count"'
-    num_lines: 13
+    num_lines: 20
```

Comment:
These are necessary because the html file that's testing against has both many lines, and newlines within those lines.